lib/registries/addon/components/license-viewer/template.hbs

```diff
@@ -1,30 +1,32 @@
 <OsfDialog as |dialog|>
-    <dialog.trigger>
-        <OsfButton
+    <div local-class='LicenseView'>
```

Comment:
I think this is still a bit wide of a net to cast on the css. Can we just focus on the field that needs the text overflow applied to it, rather than the whole dialog box and trigger?

Reply:
I have just updated the local-class name to 'LicenseName' and wrapped it around only the registration license name model on line 25 of the same file.

Reply:
I have just updated the local-class name to 'LicenseName' and wrapped it around only the registration license name model on line 10 of the same file per the recommendation below.